OperatingSystem-Services/Platform-Linux/OpenVAS/README.md

```diff
@@ -0,0 +1,419 @@
+# OpenVAS (Greenbone Vulnerability Manager)
+>[!NOTE] This install guide uses the Greenbone Community Containers. Running on VM with 2 CPUs and 4 GB ram GVM runs slow. GVM Documentation: https://greenbone.github.io/docs/latest/22.4/container/index.html
+
+## Prerequisites
+1. **Install GVM Dependencies**
+	```
+	sudo apt install ca-certificates curl gnupg
+	```
+	
+2. **Uninstall Conflicting Packages**
+	```
+	for pkg in docker.io docker-doc docker-compose podman-docker containerd runc; do sudo apt remove $pkg; done
+	```
+
+3. **Setup Docker Repository**
+	```
+	sudo install -m 0755 -d /etc/apt/keyrings
+	```
+	```
+	curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+	```
+	```
+	sudo chmod a+r /etc/apt/keyrings/docker.gpg
+	```
+	```
+	echo \
+	    "deb [arch="$(dpkg --print-architecture)" signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+		"$(. /etc/os-release && echo "$VERSION_CODENAME")" stable" | \
+		sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+	```
+	```
+	sudo apt update
+	```
```

Comment:
Maybe it would be good to do a little bullet point between each of the commands in order to give a little background of why we are doing this and what it does?

So if the commands do not work in the future we have more to work off. similar feedback for other points.